app/build.gradle

```diff
@@ -91,6 +92,7 @@ dependencies {
     implementation 'org.json:json:20220320'
     implementation 'com.google.guava:guava:31.1-jre'
     implementation 'androidx.appcompat:appcompat:1.6.0-alpha05'
+    implementation 'androidx.multidex:multidex:2.0.1'
```

Comment:
According to https://developer.android.com/studio/build/multidex, we don't need to enable multidex explicity